fastmot/utils/visualization.py

```diff
@@ -1,10 +1,17 @@
 import colorsys
 import numpy as np
 import cv2
-
+from .rect import get_center
 
 GOLDEN_RATIO = 0.618033988749895
 
+def draw_trajectories(frame, tracks):
+    for track in tracks:
```

Comment:
To align with the convention of the repo, I would implement this function for a single track and call it in `draw_tracks()` and add a parameter `show_trajectory` to avoid the duplicate for loop.

Reply:
You are right, that would make it faster!